lib/src/components/NxThreatCounter/NxThreatCounter.scss

```diff
@@ -18,7 +20,7 @@
   flex-direction: row-reverse;
   font-size: var(--nx-font-size-s);
   justify-content: flex-end;
-  height: 28px;
+  height: $threat-counter;
```

Comment:
Spurious change

Reply:
> Spurious change

Oh now I see, this is being imported into `NxVulnerabilityCounter`.  Going along with the conversation that was had in another thread, let's keep them totally separate until we decided to merge their implementations more properly.

Reply:
Sorry about that, I thought I had reverted the changes for `NxVulnerabilityCounter` too, reverted this variable in: [4654441](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/4654441ddb22dacf2666b6e22a7aede9cb5a195a)